src/index.html

```diff
@@ -26,16 +25,11 @@
         </div>
           <ul class="to-dos">    </ul>
        </div>
-
       <!-- <div >
         <p ></p>
       </div> -->
```

Comment:
Avoid leaving unused lines of code as comments in your project work as its not a professional practice

Reply:
In your JS files, kindly removed all unused lines of coeds as comment:
https://github.com/Ademola-coding/Todolist-js-best-practice/blob/db173f214f3e77c319924ff4adfb237e83f1b7dd/src/index.js#L159